server/poll.go

```diff
@@ -0,0 +1,147 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/model"
+)
+
+type Poll struct {
```

Comment:
I want to add `DataSchemaVersion` (e.g.: "V1") to `Poll` for the future.

Additionally, I think that `Poll` should have `PollID` filed, because I think that it is preferable for data to be self-describing. If DataSchemaVersion should be changed, It is easier to write function.

How do you think about this?

Reply:
I like the idea of a version. I will add it to `poll` and the api.